tests/README.md

```diff
@@ -150,6 +149,36 @@ Specify backend and image to use:
     - `hypervisor`: Typically, you will use the `qemu:///session`, or `qemu:///system`.
        It depends on where your VM resides.
     - `domain`: `libvirt` domain, which is basically name of the virtual machine.
+
+*NOTE*: It might happen, especially while using other distros than Fedora or RHEL (for example Arch), that you encounter the following error:
```

Comment:
Can you add this whole part about nested virtualization error to Admonition block? You can find how to create Admontion block at https://asciidoctor.org/docs/asciidoc-writers-guide/#admonition-blocks
(I'm not sure about readability of asciidoc documentation for you, so if you have any problem, just let me know and I will help you with fix)